service.subtitles.opensubtitles/addon.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.opensubtitles"
+       name="OpenSubtitles.org"
+       version="5.1.0"
+       provider-name="amet, opensubtitles">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+	</requires>
+	<extension point="xbmc.subtitle.module"
+             library="service.py" />
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en">OpenSubtitles</summary>
```

Comment:
Please use the correct locale identifier. E.g.: `en_GB` instead of `en`. That goes for all `lang=""` attributes.